scripts/scripts.js

```diff
@@ -337,6 +341,7 @@ async function loadEager(doc) {
     decorateMain(main);
     await loadEagerBlocks(main);
     await detectSidebar(main);
+    await h3Styling(main);
```

Comment:
Usually we put these kind of functions at the end of `decorateMain`, unless there is a good reason for them to be outside.

Reply:
moved the code to decorateMain, working fine.